_posts/2015-11-11-full-stack-es6-with-webpack-and-babel.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "On the way to full stack ECMAScript 6"
+date: "2015-11-16 12:00:00"
+icon: code
+categories: tech-stories
+authors: ["Paolo"]
+---
+
+**JavaScript is moving fast these days.** With all the languages out there compiling to it, JS itself is making a big leap forwards with the standardisation of its next version called _ECMAScript 6_ (or _ES6_, or _ECMAScript 2015_, or _ES2015_. [Naming is probably the hardest programming related task](https://twitter.com/a_ramella/status/656522461480099840), and if you think that _ECMAScript sounds like a skin disease_, you're [in good company with the inventor of JavaScript himself](https://mail.mozilla.org/pipermail/es-discuss/2006-October/000133.html)).
```

Comment:
To be honest, people who don't get scared away by the headline will definitely know what JS stands for in this context.

I adopted all the other changes, it's always a pleasure to learn from a native speaker :)